commands/info/command.go

```diff
@@ -192,7 +194,7 @@ func info(args Arguments) (infoResult, error) {
 
 	// If an endpoint and a token is defined, we pull info from the API, too.
 	if args.apiEndpoint != "" && args.token != "" {
```

Comment:
as this is the only place where `args.token` is used. What do you think about defining a boolean like `isUserProvided`?

As a different option we could also change the `client.NewWithConfig` function if that is easier than changing every single `gsctl` command.

Reply:
Thanks for the suggestion. I explored it and see it complicating things in many places. If important, let's please try that in another PR.